modules/ding_frontend/ding_frontend.features.menu_links.inc

```diff
@@ -29,6 +29,8 @@ function ding_frontend_menu_default_menu_links() {
     'expanded' => 0,
     'weight' => -47,
     'customized' => 1,
+    'language' => 'und',
+    'menu_links_customized' => 1,
```

Comment:
What does `menu_links_customized` do?

Reply:
Can't explain what this is doing, but after some investigation can say that this have relation with database field 'customized' in menu_links table. Also by default this value in the database is set to 0, and if features module had exported this as 1 then I suppose this should be there.